apps/beautifier/src/string.ts

```diff
@@ -0,0 +1,22 @@
+import { beautify } from '@etalab/adresses-util/lib/voies'
+import { trim, deburr } from 'lodash-es/lodash.js'
+// import { normalize } from '@nivalis/normadresse'
+
+function slugify(str: string) {
+    return trim(deburr(str).toLowerCase().replace(/\W+/g, '-'), ' -')
+}
+
+function beautifyUppercased(str: string) {
+    return str === str.toUpperCase()
+        ? beautify(str)
+        : str
+}
```

Comment:
La fonction 'beautifyUppercased' devrait se comporter differement en fonction de la langue du contenue qui lui est passée (la regle actuelle ne s'applique que pour le français) : 

```suggestion
export const DEFAULT_ISO_CODE = 'fra'; // Default ISO code for labels

function beautifyUppercased(str: string, lang: string = DEFAULT_ISO_CODE) {
    if(str !== str.toUpperCase()) {
        return str
    }
    
    switch(lang) {
        case DEFAULT_ISO_CODE :
            return beautify(str)
        default :
            return str
    }
}
```

NB : L'usage est donc à mettre en jour dans la fonction `getLabelsFromRow`